zerver/views/messages.py

```diff
@@ -863,6 +866,40 @@ def json_update_message(request, user_profile):
     return update_message_backend(request, user_profile)
 
 @has_request_variables
+def highlight_message_differences(message_history, sender_email):
+    # type: (List[Dict[str, Any]], str) -> List[Dict[str, Any]]
+    for edited_msg, prev_msg in zip(message_history, message_history[1:]):
+        # Iterate over message edit history
+        rendered_content = highlight_html_differences(prev_msg['prev_content'], \
+                                                      edited_msg['prev_content'])
+        edited_msg_index = message_history.index(edited_msg)
+        message_history[edited_msg_index]['prev_rendered_content'] = rendered_content
```

Comment:
Fro v1, let just highlight HTML differences rendered content(This PR is getting large). Will create another PR to add a tab switcher between rendered and unrendered content. Thoughts?